### ISSUE TITLE
SyntaxWarning raised during import

### ISSUE THREAD

**Issue by tleonardi** [titipata/pubmed_parser#75]
Importing pubmed_parser in python3 raises the following warnings:
```
Python 3.8.1 (default, Jan  8 2020, 23:09:20) 
[GCC 9.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pubmed_parser as p
/home/tleonardi/nobackup/pubmed_parse_tleonardi/pubmed_parser/pubmed_web_parser.py:147: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  pmc_text = [c for c in citation.split('/') if c is not ''][-1]
/home/tleonardi/nobackup/pubmed_parse_tleonardi/pubmed_parser/pubmed_web_parser.py:246: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if id_type is 'PMC':
/home/tleonardi/nobackup/pubmed_parse_tleonardi/pubmed_parser/pubmed_web_parser.py:249: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif id_type is 'PMID':
>>> 
```
This is due to a [documented](https://docs.python.org/3.8/whatsnew/3.8.html) change in Python3.8.
```
The compiler now produces a SyntaxWarning when identity checks (is and is not)
are used with certain types of literals (e.g. strings, numbers). These can often work 
by accident in CPython, but are not guaranteed by the language spec. The warning
advises users to use equality tests (== and !=) instead. (Contributed by Serhiy
Storchaka in bpo-34850.)
```

**Comment by titipata** [comment]
@tleonardi, ah, thanks for the catch! I will check through the repository to see if there are more errors similar to this issue before closing it.

**Comment by titipata** [comment]
I checked this issue and it seems like this should be all fixed after #78. I will close this issue for now. Thanks @tleonardi for the contribution!
